frame/oracle/src/lib.rs

```diff
@@ -457,23 +461,27 @@ pub mod pallet {
 		pub fn add_asset_and_info(
 			origin: OriginFor<T>,
 			asset_id: T::AssetId,
-			threshold: Percent,
-			min_answers: u32,
-			max_answers: u32,
-			block_interval: T::BlockNumber,
+			valid_threshold: Validated<Percent, ValidThreshhold>,
+			valid_min_answers: Validated<u32, ValidMinAnswers>,
+			valid_max_answers: Validated<u32, ValidMaxAnswer<T::MaxAnswerBound>>,
+			valid_block_interval: Validated<T::BlockNumber, ValidBlockInterval<T::StalePrice>>,
```

Comment:
I don't think there is a need to name these valid_foo to be honest. The type reflects what it is.

Reply:
Fixed in 628e2577c713fb871ca3aae8c98926b0bf9383b0